tlscanary/tools/one_crl_downloader.py

```diff
@@ -95,3 +95,53 @@ def get_list(onecrl_env, workdir, commit, use_cache=True, cache_timeout=60*60):
         logger.warning("Using cached OneCRL revocations data from `%s`" % cache_file)
 
     return cache_file
+
+# populate_cert_storage creates an LMDB database populated with the contents of the provided
+# OneCRL environment (one of either "stage" or "production"). The created database will be
+# located at <WORKDIR>/cache/<ONE_CRL_ENV>_cert_storage/security/data.safe.bin. This database
+# should in turn be copied to <FIREFOX_PROFILE>/security_state/data.safe.bin.
+def populate_cert_storage(onecrl_env, workdir, commit="master", use_cache=True, cache_timeout=60*60):
+    global logger
+
+    dc = cache.DiskCache(os.path.join(workdir, "cache"), cache_timeout, purge=True)
+    cache_id = "%s_cert_storage" % onecrl_env
+    if not use_cache:
+        # Enforce re-extraction even if cached
+        dc.delete(cache_id)
+    if cache_id in dc:
+        logger.warning("Using cached OneCRL cert_storage data from `%s`" % dc[cache_id])
+        return
+
+    cached_security_state = dc[cache_id]
+    os.makedirs(cached_security_state)
+
+    repo_dir = @TODO
```

Comment:
I need a bit of help on deciding this one. We _were_ depending on `go get` effectively being a a proxy to `git`, and thus `$GOPATH` being the path to the repo. Now, we _could_ use the `$GOPATH` as a sort of repo path, but that seems fragile given the mixed codebase now. Perhaps clone it into the cache?